modules/claims_api/app/views/claims_api/unsuccessful_report_mailer/unsuccessful_report.html.erb

```diff
@@ -30,81 +30,77 @@
+
+    <h3>
+      <%= @grouped_claims_errors.count %>
       <span class="title">Error Counts</span>
-    </h2>
-    <%= render partial: 'counts_table', locals: { statistics: @grouped_errors } unless @grouped_errors.count.zero? %>
+    </h3>
+    <%= render partial: 'counts_table', locals: { statistics: @grouped_claims_errors } unless @grouped_claims_errors.count.zero? %>
 
-    <h2>
-      <%= @grouped_warnings.count %>
+    <h3>
+      <%= @grouped_claims_warnings.count %>
       <span class="title">Warning Counts</span>
-    </h2>
-    <%= render partial: 'counts_table', locals: { statistics: @grouped_warnings } unless @grouped_warnings.count.zero? %>
+    </h3>
+    <%= render partial: 'counts_table', locals: { statistics: @grouped_claims_warnings } unless @grouped_claims_warnings.count.zero? %>
 
-    <h2>
+    <h3>
       <%= @flash_statistics.count %>
       <span class="title">Flash Counts</span>
-    </h2>
+    </h3>
     <%= render partial: 'counts_table', locals: { statistics: @flash_statistics } unless @flash_statistics.count.zero? %>
 
-    <h2>
+    <h3>
       <%= @special_issues_statistics.count %>
       <span class="title">Special Issues Counts</span>
-    </h2>
+    </h3>
     <%= render partial: 'counts_table', locals: { statistics: @special_issues_statistics } unless @special_issues_statistics.count.zero? %>
 
-    <h2>
-      <%= @pending_submissions.count %>
+    <h3>
+      <%= @pending_claims_submissions.count %>
       <span class="title">Pending Submissions</span>
-    </h2>
-    <%= render partial: 'submission_table', locals: { claims: @pending_submissions } unless @pending_submissions.count.zero? %>
+    </h3>
+    <%= render partial: 'submission_table', locals: { claims: @pending_claims_submissions } unless @pending_claims_submissions.count.zero? %>
+
+    <h3>
+      <%= @unsuccessful_claims_submissions.count %>
+      <span class="title">Errored Submissions</span>
+    </h3>
+    <%= render partial: 'submission_table', locals: { claims: @unsuccessful_claims_submissions } unless @unsuccessful_claims_submissions.count.zero? %>
+
+    <hr/>
+
+    <h2>Power of Attorney Totals</h2>
+
+    <h3>
+      <%= @poa_totals[:total] %>
+      <span class="title">POA Submissions</span>
+    </h3>
+    <%= render partial: 'poa_status_table', locals: { poa_submissions: @poa_totals } unless @poa_totals.blank? %>
 
-    <h2>
-      <%= @unsuccessful_submissions.count %>
-      <span class="title">Errorred Submissions</span>
-    </h2>
-    <%= render partial: 'submission_table', locals: { claims: @unsuccessful_submissions } unless @unsuccessful_submissions.count.zero? %>
+    <h3>
+      <%= @unsuccessful_poa_submissions.count %>
+      <span class="title">Errored Submissions</span>
+    </h3>
+    <%= render partial: 'poa_errors_table', locals: { poa_errors: @unsuccessful_poa_submissions } unless @unsuccessful_poa_submissions.count.zero? %>
```

Comment:
this is weird formatting by the git diff. This is new for `poa_errors`. _Claims_ submission errors got renamed in lines 84-88 above